docs/semantics.md

```diff
@@ -377,3 +377,21 @@ Not all of the usual file system features are supported. Most prominently:
 - File and directory permissions and ownership cannot be changed. See the permissions section above.
 - Modification times are not tracked for any inodes except for files.
 - No other times besides modification time are tracked. For example, ctime and atime are not tracked (but will be set to something reasonable). Requests to change them will appear to succeed, but the results are unspecified.
+
+**Unsupported object names**
+
+Cloud Storage objects having following constraints in their name
+are not accessible through Cloud Storage Fuse.
+
+1. `''` (empty-name), Or
+1. Names beginning with `/`, Or
+1. Names containing `//`.
+
+The above objects cannot be accessed
+through Cloud Storage Fuse, as these names have
+special meanings in linux-based filesystems.
+For example, a file/directory with path `a//b`
+is treated the same as `a/b`.
+
+Any objects having such names are invisible to
```

Comment:
Are we ignoring a//b like file names or treating them same as a/b?

Reply:
done